package.json

```
@@ -0,0 +1,91 @@
+{
+  "name": "jci-moyeo-design-system",
+  "version": "0.1.1",
+  "module": "dist/index.js",
+  "types": "dist/types/index.d.ts",
+  "files": [
+    "/dist"
+  ],
+  "dependencies": {},
+  "peerDependencies": {
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  },
+  "scripts": {
```

빌드 전 `dist` 폴더 깔끔하게 지우는거랑, npm에 publish하면 `yarn build` 되게 스크립트를 추가하는 것도 괜찮을 것 같아요
```suggestion
  "scripts": {
     "prebuild": "rm -rf dist",
     "prepublishOnly": "yarn build",
```